packages/manager/apps/container/package.json

```
@@ -35,10 +35,11 @@
     "@ovh-ux/request-tagger": "^0.2.0",
     "@ovh-ux/shell": "^3.5.0",
     "@ovh-ux/ui-kit": "^6.7.8",
-    "@ovhcloud/ods-core": "^14.0.1",
-    "@ovhcloud/ods-stencil": "^14.0.1",
-    "@ovhcloud/ods-theme-blue-jeans": "^14.0.1",
-    "@ovhcloud/ods-theming": "^14.0.1",
+    "@ovhcloud/ods-common-core": "^17.2.1",
+    "@ovhcloud/ods-common-testing": "^17.2.1",
+    "@ovhcloud/ods-common-theming": "^17.2.1",
+    "@ovhcloud/ods-components": "^17.2.1",
+    "@ovhcloud/ods-theme-blue-jeans": "^17.2.1",
```

```suggestion
    "@ovhcloud/ods-common-core": "17.2.1",
    "@ovhcloud/ods-common-testing": "17.2.1",
    "@ovhcloud/ods-common-theming": "17.2.1",
    "@ovhcloud/ods-components": "17.2.1",
    "@ovhcloud/ods-theme-blue-jeans": "17.2.1",
```

To avoid breaking changes in the future